File: app/src/main/java/ru/magflayer/spectrum/presentation/common/android/navigation/router/GlobalRouter.java
package ru.magflayer.spectrum.presentation.common.android.navigation.router;

import ru.magflayer.spectrum.presentation.common.android.navigation.Screens;
import ru.terrakok.cicerone.Router;

public class GlobalRouter extends Router {

    public void startSplash() {
        replaceScreen(Screens.SPLASH_ACTIVITY_SCREEN);
    }

    public void startMain() {
        replaceScreen(Screens.MAIN_ACTIVITY_SCREEN);
    }

}

File: app/src/main/java/ru/magflayer/spectrum/presentation/common/android/navigation/router/GlobalRouter.kt
package ru.magflayer.spectrum.presentation.common.android.navigation.router

import com.github.terrakok.cicerone.Router
import ru.magflayer.spectrum.presentation.common.android.navigation.GlobalScreens

class GlobalRouter : Router() {

    fun startSplash() = replaceScreen(GlobalScreens.Splash)

    fun startMain() = replaceScreen(GlobalScreens.Main)

}
